devops/sql/init.sql

```diff
@@ -18,7 +18,7 @@ CREATE TABLE studies(
 
 CREATE TABLE study_confs(
        created TIMESTAMP DEFAULT CURRENT_TIMESTAMP(),
-       study_id UUID NOT NULL REFERENCES studies(id),
+       study_id UUID NOT NULL REFERENCES studies(id) ON DELETE CASCADE,
```

Comment:
@nandanrao im assuming if a study is deleted we want to cleanup any related config.

Reply:
Yeah... Need to come up with migrations at some point...